permissions/component-plugins-compat-tester-aggregator.yml

```diff
@@ -3,6 +3,8 @@ name: "plugins-compat-tester-aggregator"
 github: "jenkinsci/plugin-compat-tester"
 paths:
 - "org/jenkins-ci/tests/plugins-compat-tester-aggregator"
+cd:
```

Comment:
needs to be combined in one file for multi module to work iirc

Reply:
Mm, maybe. #1747. @daniel-beck any idea? Would it suffice as a workaround to just add a `cd` section to _one_ component file?

Reply:
https://ci.jenkins.io/job/Infra/job/repository-permissions-updater/job/PR-2103/1/artifact/json/cd.index.json looks OK.

Reply:
While https://github.com/jenkins-infra/repository-permissions-updater/blob/fe278d5176de193e3677d605af26d3e69b32808c/src/main/groovy/io/jenkins/infra/repository_permissions_updater/ArtifactoryPermissionsUpdater.groovy#L89-L95 captures each YAML file associated with a given repo and `cd.enabled`, it seems from https://github.com/jenkins-infra/repository-permissions-updater/blob/fe278d5176de193e3677d605af26d3e69b32808c/src/main/groovy/io/jenkins/infra/repository_permissions_updater/ArtifactoryPermissionsUpdater.groovy#L222-L236 and https://github.com/jenkins-infra/repository-permissions-updater/blob/fe278d5176de193e3677d605af26d3e69b32808c/src/main/groovy/io/jenkins/infra/repository_permissions_updater/ArtifactoryPermissionsUpdater.groovy#L247 that the values are actually unused. IOW it should not even matter—so long as at least one file related to this repo enables CD, we should get the right user and secrets created—though it would have been clearer for https://github.com/jenkins-infra/repository-permissions-updater/blob/fe278d5176de193e3677d605af26d3e69b32808c/src/main/groovy/io/jenkins/infra/repository_permissions_updater/ArtifactoryPermissionsUpdater.groovy#L61 to be a `TreeSet<String>`.